File: Problem 48/solution48.py
#! /usr/bin/env python
import sys

if __name__ == "__main__":
   sum = 0
   for i in range(1, int(sys.argv[1]) + 1):
      num = i
      for j in range(1, i):
         num *= i
      sum += num
   print sum
  

File: Problem 48/solution48.cc
#include <iostream>
#include "../../Integer/Integer.h"

using namespace std;

int main(int argc, char **argv) {
   Integer sum, num;

   for (int i = 1; i <= atoi(argv[1]); i++) {
      num = i;
      for (int j = 1; j < i; j++) {
         num *= i;
      }
      sum += num;
   }
   
   cout << endl << sum << endl;
   return EXIT_SUCCESS;
}
